easybuild/easyconfigs/h/Hypre/Hypre-2.14.0-foss-2018a.eb

```
@@ -9,9 +9,9 @@ description = """Hypre is a library for solving large, sparse linear systems of
 toolchain = {'name': 'foss', 'version': '2018a'}
 toolchainopts = {'pic': True}
 
-source_urls = ['https://github.com/LLNL/hypre/archive/']
+source_urls = ['https://github.com/hypre-space/hypre/archive/']
 sources = ['v%(version)s.tar.gz']
-checksums = ['10cfcd555618137c194958f84f44724fece45b58c59002d1195fed354e2ca16c']
+checksums = ['705a0c67c68936bb011c50e7aa8d7d8b9693665a9709b584275ec3782e03ee8c']
```

We should not remove the old checksum as there might be users that already have the old `v2.14.0.tar.gz` tarball and still might want to install this easyconfig of Hypre
```suggestion
checksums = [('10cfcd555618137c194958f84f44724fece45b58c59002d1195fed354e2ca16c',
              '705a0c67c68936bb011c50e7aa8d7d8b9693665a9709b584275ec3782e03ee8c')]
```